src/content/en/updates/2017/12/chrome-64-media-updates.md

```diff
@@ -0,0 +1,349 @@
+
+## Persistent licenses for Windows and Mac {: #persistent-licenses-windows-mac }
+
+Persistent license in [Encrypted Media Extensions (EME)] means the license can
+be persisted on the device so that applications can load the license into
+memory without sending another license request to the server. This is how
+offline playback is supported in EME.
+
+Until now, Chrome OS and Android were the only platforms to support persistent
+licenses. It is not true anymore. Playing protected content through EME while
+the device is offline is now possible on Windows and Mac as well.
+
+```js
+const config = [{
+  sessionTypes: ['persistent-license'],
+  videoCapabilities: [{
+    contentType: 'video/webm; codecs="vp9"',
+    robustness: 'SW_SECURE_DECODE' // Widevine L3
+  }]
+}];
+
+navigator.requestMediaKeySystemAccess('com.widevine.alpha', config)
+.then(access => {
+  // User will be able to watch encrypted content while being offline when
+  // license is stored locally on device and loaded later.
+})
+.catch(error => {
+  // Persistent licenses are not supported on this platform yet.
+});
+```
+
+You can try persistent licenses yourself by checking out the [Sample Media PWA]
+and following these steps:
+
+1. Go to [https://biograf-155113.appspot.com/ttt/episode-2/]
+2. Click "Make available offline" and wait for the video to be downloaded.
+3. Turn off internet connection.
+4. Click the "Play" button and enjoy the video!
+
+## Media preload defaults to "metadata" {: #media-preload-defaults-metadata }
+
+Matching other browsers implementations, Chrome desktop now sets the default
+preload value for `<video>` and `<audio>` elements to "metadata" in order to
+reduce bandwidth and resource usage. This new behaviour only applies to cases
+where no preload value is set. Note that the preload attribute's hint is
+discarded when a `MediaSource` is attached to the media element as the web site
+handles its own preload.
+
+In other words, `<video>` preload value is now "metadata" while `<video
+preload="auto">` preload value stays "auto". Give a try to the [official
+sample](https://googlechrome.github.io/samples/media/preload-metadata).
+
+[Intent to Ship](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/5CDvJkdxyQ8) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5682169347309568) &#124;
+[Chromium Bug](https://crbug.com/310450)
+
+## Unsupported playbackRate raises an exception {: #unsupported-playbackRate-raises-exception }
+
+Following an [HTML specification change], when media elements' playbackRate
+is set to a value not supported by Chrome (eg. negative value), a
+"NotSupportedError" DOMException is thrown in Chrome 63.
+
+```js
+const audio = document.querySelector('audio');
+try {
+  audio.playbackRate = -1;
+} catch(error) {
+  console.log(error.message); // Failed to set the playbackRate property
+}
+```
+
+For info, Chrome’s current implementation raises this exception when new
```

Comment:
What do you mean by 'when new playbackRate'?

Reply:
I've removed the `new` as it was confusing. I meant the new value set by user.